spring-cloud-starter-stream-sink-jdbc/src/main/java/org/springframework/cloud/stream/app/jdbc/sink/JdbcSinkConfiguration.java

```diff
@@ -63,7 +80,10 @@
  * @author Eric Bottard
  * @author Thomas Risberg
  * @author Robert St. John
+ * @author Oliver Flasch
  */
+@Configuration
+@EnableScheduling
```

Comment:
Please, consider to use `groupTimeout` functionality on the aggregator instead of this and `MessageGroupStoreReaper`.
This way each group will decide when to release itself instead of one centralized coordination.
See more info in the Docs: https://docs.spring.io/spring-integration/reference/html/messaging-routing-chapter.html#agg-and-group-to

Reply:
Many thanks for the quick and useful comment. I changed this PR as requested. I which you a nice holiday season!